README.md

```diff
@@ -2,21 +2,36 @@
 
 # cardano-benchmarking
 
-## preparation
+This repository contains two important tools for `cardano-node` benchmarking:
+
+1. `cardano-tx-generator`
+2. `cardano-rt-view-service`
+
+The first one is a generator of transactions, both for Byron and Shelley. For more details please [see the project](https://github.com/input-output-hk/cardano-benchmarking/tree/master/cardano-tx-generator).
+
+The second one is a service that allows to see the state of running `cardano-node` processes in the real time. For more details please [see the project](https://github.com/input-output-hk/cardano-benchmarking/tree/master/cardano-rt-view).
+
```

Comment:
we need a small (250x180?) screenshot here..

Reply:
Yes, agree, we need a screenshot. But probably not here, but in `README.md` of `cardano-rt-view`?

Reply:
Added to `README.md` in `cardano-rt-view`.